lib/src/components/NxToast/__tests__/NxToastContainer.test.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+import React from 'react';
+import { rtlRenderElement, rtlRender } from '../../../__testutils__/rtlUtils';
+import { screen, render, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import NxToastContainer from '../NxToastContainer';
+import { NxToastContainerProps } from '../types';
+import NxToast from '../NxToast';
+
+import NxAlert from '../../NxAlert/NxAlert';
+import { faEye } from '@fortawesome/free-solid-svg-icons';
+
+describe('NxToastContainer', function() {
+  const simpleProps: NxToastContainerProps = {
+    children: (
+      <>
+        <NxToast onClose={()=>{}}>
+          <NxAlert icon={faEye}>This is an Alert</NxAlert>
+        </NxToast>
+        <NxToast onClose={()=>{}}>
+          <NxAlert icon={faEye}>This is an Alert</NxAlert>
+        </NxToast>
+        <NxToast onClose={()=>{}}>
+          <NxAlert icon={faEye}>This is an Alert</NxAlert>
+        </NxToast>
+      </>
+    )
+  };
+
+  const quickRender = rtlRender(NxToastContainer, simpleProps);
+  const renderEl = rtlRenderElement(NxToastContainer, simpleProps);
+
+  it('renders any NxToasts passed into it', function() {
+    renderEl();
+
+    const alerts = screen.getAllByText('This is an Alert');
+
+    expect(alerts.length).toBe(3);
+  });
+
+  it('sets focus to last toast when first toast is closed', async function() {
```

Comment:
```suggestion
  it('sets focus to last toast\s close button when first toast is closed', async function() {
```

Reply:
Changed the name on both tests to reflect 'last toast's close button' instead of just toast here: [c158bb7](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/c158bb70dd6e54f0022e400d5ee0b1706b214adc)